### ISSUE TITLE
Where is label of Object-365-instance segmentation

### ISSUE THREAD

**Issue by ArsenLuca** [lxtGH/OMG-Seg#6]
as the title states

**Comment by lxtGH** [comment]
We use SAM-Huge to provide the mask label. Currently, we find the pre-training of such dataset only bring a little improvement (within 1% PQ on COCO dataset, less than 0.5% PQ on Cityscapes). You can also directly use CLIP-pretrained backbone for training OMG-Seg.
